### ISSUE TITLE
Plugin seems to disable formatting inside link text

### ISSUE THREAD

**Issue by claremacrae** [samwarnick/obsidian-simple-embeds#18]
Hello, thank you for a very nice plugin...

I just wanted to report an odd display problem I had, that via bisecting, I tracked down to this plugin.

I am using this markdown for the demo:

```
[samwarnick/**obsidian-simple-embeds**](https://github.com/samwarnick/obsidian-simple-embeds)
```

## Expected behaviour

With these settings...

- **no plugins**
- with default theme
- no snippets
- legacy editor (no live preview)

... the previewed display in Obsidian 0.13.15 is as expected:

![image](https://user-images.githubusercontent.com/4840096/147781415-eb74c4af-113a-404c-aad2-c27000ccf28d.png)

## Actual behaviour

With these settings, however...

- **only Simple Embeds 1.6.1 installed**
- with default theme
- no snippets
- legacy editor (no live preview)

... the previewed display in Obsidian 0.13.15 is missing the bold text:

![image](https://user-images.githubusercontent.com/4840096/147781595-a14b4d81-13ee-40b4-a12e-a7f9113fa012.png)

In other words, the formatting seems to have been lost.

The behaviour is the same if I turn Live Preview on (I turned it off for the purposes of bug-reporting)

**Comment by samwarnick** [comment]
Thanks for letting me know and the excellent write-up. I think I know what's going on here, so hopefully won't be too difficult to fix. I'll get to this soon.

**Comment by claremacrae** [comment]
Super - thanks very much indeed!

**Comment by claremacrae** [comment]
Confirmed that this is fixed via 1.6.2, after Obsidian is restarted.

Very many thanks for fixing it!
